### PR TITLE
Website header

### PR DESCRIPTION
**Pull request by Medrine-Jepkemoi**
# Description

I have created the header which includes: 

1. "Elewa" logo on the left
2. A transparent background 
3. Black text in correct font 
4. Contact button on the right

This is issue #1

## Type of change

Please delete options that are not relevant.

-  Bug fix (non-breaking change which fixes an issue)
-  New feature (non-breaking change which adds functionality)

## Screenshot (optional)
Screenshot 2023-10-03 at 17 20 39

## Checklist

- My code follows the style guidelines of this project
-  I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
-  My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
-  Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/css/style.css
Comment (paulkahura) [inline src/css/style.css:1]: use rem/em over px

**Review by paulkahura — CHANGES_REQUESTED** [review]
Nice attempt however I'll need the mobile and tablet screenshots to review.

**Review by JAPHETHNYARANGA — COMMENTED** [review]
closing as this is done

**Review by JAPHETHNYARANGA — COMMENTED** [review]
closing as this is already done

**Pull request closed without merging**